docs/commerce/shipping_management/extend_shipping.md

```diff
@@ -0,0 +1,157 @@
+```
+
+Then, register the storage converter as a service:
+
+``` yaml
+[[= include_file('code_samples/front/shop/shipping/config/packages/services.yaml', 0, 1) =]][[= include_file('code_samples/front/shop/shipping/config/packages/services.yaml', 25, 28) =]]
+```
+
+#### Storage definition 
+
+Now, create a storage definition class and a corresponding schema.
+The table stores information specific for the custom shipping method type.
+
+!!! note "Create table"
+
+    Before you can proceed, in your database, create a table that has columns present in the storage definition, for example:
+
+    `CREATE TABLE ibexa_shipping_method_region_custom(id int auto_increment primary key, customer_id text, shipping_method_region_id int);`
+
+In `src/ShippingMethodType/Storage`, create `StorageDefinition.php` and `StorageSchema.php` files with their corresponding classes.
+
+``` php
+[[= include_file('code_samples/front/shop/shipping/src/ShippingMethodType/Storage/StorageDefinition.php') =]]
+```
+
+``` php
+[[= include_file('code_samples/front/shop/shipping/src/ShippingMethodType/Storage/StorageSchema.php') =]]
+```
+
+Then, register the storage definition as a service:
+
+``` yaml
+[[= include_file('code_samples/front/shop/shipping/config/packages/services.yaml', 0, 1) =]][[= include_file('code_samples/front/shop/shipping/config/packages/services.yaml', 28, 31) =]]
+```
+
+![Creating a shipping method of custom type](img/custom_shipping_method_type.png "Creating a shipping method of custom type")
+
+## Toggle shipping method type availability
+
+When you implement a web store, you can choose if a certain shipping method is available for selection during checkout.
+Here, you limit shipping method availability to customers who meet a specific condition.
+In `src/ShippingMethodType/Voter`, create a `CustomVoter.php` file with a voter class:
+
+``` php
+[[= include_file('code_samples/front/shop/shipping/src/ShippingMethodType/Voter/CustomVoter.php') =]]
+```
+
+Register the voter as a service:
+
+``` yaml
+[[= include_file('code_samples/front/shop/shipping/config/packages/services.yaml', 0, 1) =]][[= include_file('code_samples/front/shop/shipping/config/packages/services.yaml', 31, 34) =]]
+```
+
+## Display shipping method parameters in shipping method details view
+
+You can extend the default  shipping method details view by making shipping method visible on the **Cost** tab.
+To do this, in `src/ShippingMethodType/Cost`, create a `CustomCostFormatter.php` file with a formatter class:
+
+``` php
+[[= include_file('code_samples/front/shop/shipping/src/ShippingMethodType/Cost/CustomCostFormatter.php') =]]
+```
+
+Then register the formatter as a service:
+
+``` yaml
+[[= include_file('code_samples/front/shop/shipping/config/packages/services.yaml', 0, 1) =]][[= include_file('code_samples/front/shop/shipping/config/packages/services.yaml', 34, 38) =]]
+```
+
```

Comment:
We could use a screenshot here.
The whole PR after doc review should go through QA.

Reply:
Already asked on a proper channel but there was no response. Will try again